lib/confluence_table.rb

```
@@ -0,0 +1,72 @@
+    @page_id = page_id
+    @api_token = api_token
+    @username = username
+    @auth_header = "Basic #{Base64.strict_encode64("#{@username}:#{@api_token}")}"
+  end
+
+  def fetch_page
+    url = URI("#{BASE_URL}#{@page_id}?body-format=storage")
+    request = Net::HTTP::Get.new(url)
+    request["Authorization"] = @auth_header
+    response = send_request(url, request)
+    json = response.body
```

rubocop

[warning] Useless assignment to variable - `json`.

(Lint/UselessAssignment)